src/utils/spacing.css

```diff
@@ -0,0 +1,52 @@
+/* Array of sizing values in REM */
+@each $val, $i in 0.25, 0.5, 0.75, 1, 1.5, 2, 3, 4, 5, 10 {
```

Comment:
I feel this is kinda defeating the purpose of the design tokens. We are now duplicating these values. We should probably come up with a better solution for this.

Also the `0.25` value won't work due to the `.`. I think we should go with an index system. So we get `1, 2, 3, 4` etc and they map to something in the array you have now.

Reply:
@xadamy Do you have tips on how we can do this?

Reply:
For now we decided to generate these using Tailwind, and go from there.